deliverables/ruby_monk_problems/number_shuffle.rb

```diff
@@ -1,3 +1,21 @@
+def fact(n)
+  if n<= 1
+    1
+  else
+    n * fact( n - 1 )
+  end
+end
+
 def number_shuffle(number)
-  # your code here
+  out = []
+  n_fact = fact(number.to_s.split('').length)
```

Comment:
Awesome.  You built a more generalized solution instead of just using 66 or 24.  Good job!